### PR TITLE
Users improvements

### PR DESCRIPTION
**Pull request by sparksofl**
Added all of requested features
Probably later there will be design improvements (depends on the comments)

### CODE REVIEW

File: app/views/users/show.html.slim [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+      .speaker-label Speaker
+    .info
+      - unless @user.title.nil?
+        p #{@user.title}
+      - unless @user.work_since.nil?
+        p Started working #{distance_of_time_in_words(@user.work_since, DateTime.now.utc, include_seconds: false)} ago
     br
+    div.container
+      h4 About
+      - unless @user.desc.empty?
```
Comment (brytannia) [inline app/views/users/show.html.slim]: do not use unless with else, better to use if else

File: app/views/users/show.html.slim [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+        p No information yet
     = link_to 'Edit profile', edit_user_path(current_user)
+  div
+    .container.info
+      h3 Contacts
+      = image_tag "mail.png"
+      = @user.email
+      - unless @user.phone.nil?
+      br
+        = image_tag "phone.png"
```
Comment (brytannia) [inline app/views/users/show.html.slim]: in such cases you can use some icon fonts such as https://design.google.com/icons/ or https://fortawesome.github.io/Font-Awesome/

**Comment by brytannia** [comment]
@sparksofl good job! Add some minor comments, please, look at it and ping me where you think this PR is ready to merge.

**Comment by sparksofl** [comment]
@brytannia closed another issue. If everything is fine with last commits, PR is ready to merge.

File: app/views/users/show.html.slim [hunk 3]
```diff
@@ -1,12 +1,29 @@
...
+      h4 About
+      - if !@user.desc.blank?
+        p #{@user.desc}
+      - else
+        p No information yet
     = link_to 'Edit profile', edit_user_path(current_user)
+  div
+    .container.info
+      h3 Contacts
+      i class="fa fa-envelope-o"
```
Comment (brytannia) [inline app/views/users/show.html.slim:24]: :+1:

**Pull request merged**